db/seeds.rb

```diff
@@ -9,3 +9,9 @@
   User.create! name: name, email: email,password: password,
     password_confirmation: password, activated: true, activated_at: Time.zone.now
 end
+
+users = User.order(:created_at).take(6)
+50.times do
+  content = Faker::Lorem.sentence(5)
+  users.each { |user| user.microposts.create!(content: content) }
+end
```

Comment:
[No comment] Hỏi về cái hình bên dưới thôi, sao search "dog" mà lại ra toàn "goat" thế (yaoming)